docker/brew.Dockerfile

```diff
@@ -128,6 +132,11 @@ RUN $YARN install --frozen-lockfile --production --network-timeout 600000
 FROM registry.redhat.io/ubi9/nodejs-18-minimal:1 AS runner
 USER 0
 
+RUN microdnf update -y && \
+  microdnf install -y python3 python3-pip && \
+  pip3 install mkdocs-techdocs-core==1.* && \
```

Comment:
this won't work downstream -- I'll draft you something as a replacement based on https://github.com/redhat-developer/devspaces-images/blob/devspaces-3-rhel-8/devspaces-udi/Dockerfile#L226-L229 and https://github.com/redhat-developer/devspaces-images/blob/devspaces-3-rhel-8/devspaces-udi/container.yaml#L33-L40

Reply:
Thanks for noticing this! Let's do that in a followup PR, sure.